custom_modules/MongoDbController/MongoControllerHelpers.js

```diff
@@ -169,6 +169,80 @@ class MongoControllerHelpers
 
 
 
+    /* 
+     * PATCHES
+     */
+
+    static async updateOne({
+        connection,
+        findParams,
+        obj,
+        collectionName,
+        Model,
+    })
+    {
+        return new Promise(function (resolve, reject)
+        {
+            connection.getCollection({ collectionName })
+            .then(async function (collection)
+            {
+                findParams = MongoControllerHelpers.convertIdToObjectId(findParams);
+
+                // Make query
+                const validationModel = MongoControllerHelpers.getAsModel(obj, Model);
+
+                // Validation is successful or there is no validation
+                if (!validationModel.isValid || validationModel.isValid())
+                {
+                    // Update (replace the given values for the obj)
+                    const result = await collection.findOneAndUpdate(findParams, {
+                        $set: obj,
+                    });
+
+                    // Done updating, close connection
+                    await connection.close();
+
+                    // Failed query (only happens in findOne)
+                    if (!result || !result.value)
+                    {
+                        const errResults = new MongoResults({
+                            error: `No ${Model.name} was found`,
+                            status: 500,
+                        });
+                        reject(errResults);
+                    }
+                    
+                    // Parse into model
+                    const oldModel = MongoControllerHelpers.getAsModel(result.value, Model);
+
+                    // Add any fields to newModel that weren't changed for output
+                    const newModel = MongoControllerHelpers.addFieldsFromOneModelToOther({
```

Comment:
You won't always update every single field. Therefore, those fields stay the same. So, we need to say "add any fields that didn't change to the output", so the output shows the unchanged values alongside the new values.